src/metabase/server/middleware/security.clj

```diff
@@ -69,9 +69,7 @@
                   :style-src    ["'self'"
                                  "'unsafe-inline'"
                                  "https://accounts.google.com"]
-                  :font-src     ["'self'"
-                                 (when config/is-dev?
-                                   "localhost:8080")]
+                  :font-src     ["*"]
```

Comment:
@adam-james-v Maybe we should parse domains from `application-font-files` and add them here rather than using `*`